core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -1416,6 +1421,11 @@ private void handleErrorInSyncContext(Status error) {
     SubchannelImpl(CreateSubchannelArgs args, LbHelperImpl helper) {
       this.args = checkNotNull(args, "args");
       this.helper = checkNotNull(helper, "helper");
+      subchannelLogId = InternalLogId.allocate("Subchannel", /*details=*/ authority());
+      subchannelTracer = new ChannelTracer(
```

Comment:
ChannelTracer is supposed to be encapsulated by the Logger, not exposed.

Reply:
`InternalSubchannel` needs to access `ChannelTracer` for updating `ChannelStats`, while on the other hand `ChannelLogger` takes in a `ChannelTracer`. So we need to have it be here. Existing code also does this.